spec/javascripts/app/utils/ms-formatter_spec.js

```diff
@@ -0,0 +1,13 @@
+import msFormatter from 'app/utils/ms-formatter';
+
+describe('#msFormatter', () => {
+  it('formats milliseconds as X::XX', () => {
+    const output = msFormatter(0);
+    expect(output).to.match(/\d{1}:\d{2}/);
```

Comment:
Would it be easier to just test against a string? a regex for a single value seems not as clear to me?